types/dragino/lds03a/uplink.js

```diff
@@ -0,0 +1,105 @@
+function getzf(cNum) {
+  if (parseInt(cNum) < 10) {
+    cNum = `0${cNum}`;
+  }
+
+  return cNum;
+}
+
+function getMyDate(str) {
+  let cDate;
+  if (str > 9999999999) {
```

Comment:
str does not seem to be a string but a number, also possibly rename this method to a more speaking name

Reply:
it is a string. The action call is:
`getMyDate(
    (
      (bytes[7 + i] << 24) |
      (bytes[8 + i] << 16) |
      (bytes[9 + i] << 8) |
      bytes[10 + i]
    ).toString(10),`
    
  Ps: its not my function but i could change it